govdelivery-subscribe/error/index.html

```diff
@@ -11,13 +11,13 @@
 {% block content_main %}
     <h1><span class="cf-icon cf-icon-error-round"></span> Not just yet!</h1>
     <p class="short-desc">
-        We’re still building the email signup function for <br>
-        this beta release. It will be available soon. For now:
+        We couldn’t complete your signup request. Try<br>
```

Comment:
While extremely fringe, I realized today this error page will also be displayed if the user doesn't enter an email address. Do we want to point out that possibility here?